src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -44,13 +44,24 @@ void UprobesUnwindingVisitor::visit(StackSamplePerfEvent* event) {
   const std::vector<unwindstack::FrameData>& libunwindstack_callstack = unwinder_.Unwind(
       current_maps_.get(), event->GetRegisters(), event->GetStackData(), event->GetStackSize());
 
+  // LibunwindstackUnwinder::Unwind signals an unwinding error with an empty callstack.
   if (libunwindstack_callstack.empty()) {
     if (unwind_error_counter_ != nullptr) {
       ++(*unwind_error_counter_);
     }
     return;
   }
 
+  // Callstacks with only one frame (the sampled address) are also unwinding errors, that were not
```

Comment:
What is if we have a sample inside main?

Reply:
This is not the case of main because there are other frames outside of main.

Reply:
I see. Could you add that concrete statement to the comment? Otherwise, LGTM.

Reply:
Good point, done.